pkg/api/register.go

```diff
@@ -78,12 +78,12 @@ func init() {
 
 	// Register Unversioned types
 	// TODO this should not be done here
-	Scheme.AddKnownTypes(SchemeGroupVersion, &unversioned.ListOptions{})
-	Scheme.AddKnownTypes(SchemeGroupVersion, &unversioned.Status{})
-	Scheme.AddKnownTypes(SchemeGroupVersion, &unversioned.APIVersions{})
-	Scheme.AddKnownTypes(SchemeGroupVersion, &unversioned.APIGroupList{})
-	Scheme.AddKnownTypes(SchemeGroupVersion, &unversioned.APIGroup{})
-	Scheme.AddKnownTypes(SchemeGroupVersion, &unversioned.APIResourceList{})
+	Scheme.AddKnownTypes(unversioned.SchemeGroupVersion, &unversioned.ListOptions{})
```

Comment:
I see. I think the solution is moving the Scheme out of `api`, perhaps we can make a it the `pkg/api/scheme` package. What do you think?

Surely we don't want to do the move in this PR. Can you revert this bit to the old code, and leave a TODO in the unversioned/register.go?

I'll leave a note in #16062, which is the issue for restructure of the API types.